nsds_lab_to_nwb/components/stimulus/mark_manager.py

```diff
@@ -11,10 +11,11 @@ def __init__(self, dataset):
     def get_mark_track(self, name='recorded_mark'):
         # Read the mark track
         if hasattr(self.dataset, 'htk_mark_path'):
-            mark_path = self.dataset.htk_mark_path
-            mark_track, rate = HTKReader.read_htk(mark_path)
+            mark_file = HTKFile(self.dataset.htk_mark_path)
+            mark_track, meta = mark_file.read_data()
+            rate = mark_file.sample_rate
         else:
-            mark_track, meta = TDTReader(self.dataset.tdt_path).get_data('mrk1')
+            mark_track, meta = TDTReader(self.dataset.tdt_path).get_data('mrk1', None)
```

Comment:
If the second input `dev_conf` is made a default kwarg, no need to pass `None` here

Reply:
Done